src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,38 @@
 package org.example;
 
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import org.example.database.DatabaseConnection;
+import org.example.database.DatabaseInterface;
+import org.example.model.Item;
+import org.example.services.ItemConsumer;
+import org.example.services.ItemProducer;
+import org.example.services.TaxCalculatorService;
+
 public class Main {
   public static void main(String[] args) {
+    BlockingQueue<Item> itemQueue = new LinkedBlockingQueue<>();
+    BlockingQueue<Item> processedItemQueue = new LinkedBlockingQueue<>();
```

Comment:
write about in pr description `LinkedBlockingQueue`

Reply:
Added description for LinkedBlockingQueue in pr